elementary/messages/formats/text.py

```
@@ -0,0 +1,140 @@
+        self._icon_style = icon_style
+        self._table_style = table_style
+
+    def format_icon(self, icon: Icon) -> str:
+        if self._icon_style == IconStyle.OMIT:
+            return ""
+        elif self._icon_style == IconStyle.UNICODE:
+            return ICON_TO_HTML[icon]
+        elif self._icon_style == IconStyle.NAME:
+            return f":{icon.value}:"
+        else:
+            raise ValueError(f"Invalid icon style: {self._icon_style}")
```

Pylint (3.3.7)

[refactor] Unnecessary "elif" after "return", remove the leading "el" from "elif"

(R1705)